tensorflow/tools/tf_windows_build_dockerfiles/Dockerfile

```diff
@@ -0,0 +1,132 @@
+# This Dockerfile creates an image that:
+# - Has the correct MTU setting for networking from inside the container to work.
+# - Has Visual Studio 2019 Build Tools installed.
+# - Has MSVC 14.29 and clang 14.0 installed
+# - Has msys2 + git, curl, zip, unzip installed.
+# - Has Python 3.9 installed.
+# - Has Bazelisk 0.11 installed.
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+SHELL ["powershell.exe", "-ExecutionPolicy", "Bypass", "-Command", "$ErrorActionPreference='Stop'; $ProgressPreference='SilentlyContinue'; $VerbosePreference = 'Continue';"]
+
+RUN netsh interface ipv4 set subinterface \"vEthernet (Ethernet)\" mtu=1460 store=persistent
+
+RUN md C:\TEMP
+RUN md C:\TMP
+
+RUN (New-Object Net.WebClient).DownloadFile('https://aka.ms/vs/16/release/vs_community.exe', 'C:\TEMP\vs_community.exe');
+# Install Visual Studio 2019 Build Tools.
+SHELL ["cmd", "/S", "/C"]
+RUN C:\TEMP\vs_community.exe \
+        --quiet --wait --norestart --nocache \
+        --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 \
+        --add Microsoft.VisualStudio.Workload.NativeDesktop \
+        --add Microsoft.VisualStudio.Component.VC.14.29.x86.x64 \
+        --add Microsoft.VisualStudio.Component.Windows10SDK.20348 \
+ || IF "%ERRORLEVEL%"=="3010" EXIT 0
+
+SHELL ["powershell.exe", "-ExecutionPolicy", "Bypass", "-Command", "$ErrorActionPreference='Stop'; $ProgressPreference='SilentlyContinue'; $VerbosePreference = 'Continue';"]
+
+# Install 7-Zip and add it to the path.
+RUN (New-Object Net.WebClient).DownloadFile('https://www.7-zip.org/a/7z2201-x64.msi', '7z.msi'); \
```

Comment:
Can we update this?

Reply:
The new version available is 7-Zip 23.00 (beta). Would it be advisable to proceed with installing this beta version?